migrations/create_webmaker_db.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS "users"
+(
+  id serial NOT NULL,
+  username varchar(20) NOT NULL,
+  created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
We probably want a trigger to update this dynamically: http://www.revsys.com/blog/2006/aug/04/automatically-updating-a-timestamp-column-in-postgresql/